src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(/images/star.svg);
+  background-size: contain;
```

Comment:
Using `contain` might not be the best choice; it resizes the background image to fit the parent while keeping it fully visible, resulting in larger stars than in the design.

Reply:
done :)